river/ensemble/boosting.py

```diff
@@ -1,12 +1,17 @@
 import collections
+import copy
 import math
 import typing
 
+import numpy as np
+
 from river import base, linear_model
+from river.base import DriftDetector
+from river.drift import DDM
```

Comment:
Avoid using absolute imports. It's more Pythonic to import `base` and `drift` and then access the classes you want.

Reply:
Solved in BaseModel + other changes